docs/spring-boot-support.md

```diff
@@ -0,0 +1,12 @@
+## Spring Boot Support
+
+The springBootUtility command stores the dependent library JARs of the application to the target library cache and packages the remaining application artifacts into a thin application JAR. When you specify a read-only parent library cache, the command creates a target library cache that contains only the libraries that are not available in the parent cache. Use this capability to create efficient Docker layers for your Spring Boot application.
```

Comment:
I know gradle is different than Maven, but this paragraph is so different from what is provided in liberty-maven-plugin: https://github.com/OpenLiberty/ci.maven/blob/master/docs/spring-boot-support.md#spring-boot-support

Don't we need some of that same info? And an example, and reference to the spring-boot features that need to be configured in server.xml?

Reply:
Agreed. I think this section should be more along the lines of the Maven docs rather than our Ant docs. 

It'd also be important to mention that the Spring Boot 1.5 plugin is not supported in Gradle versions 5.0 and on.